sale_order_customer_no_autofollow/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Sale order partner no autofollow",
```

Comment:
mane should follow CamelCase convention.
Like "Sale Order Customer No Autofollow"

Check any other module, eg [this one](https://github.com/OCA/sale-workflow/blob/1762ec0dde97dee449291d44c6a46cdcccfee845/sale_automatic_workflow/__manifest__.py#L7)

Reply:
Got it, I'll fix it